### PR TITLE
Add raspbian distribution

### PR DESCRIPTION
**Pull request by lannic**
## What does this PR do?
Added feature : 
Use script for updating the raspbian distribution

Add raspbian distribution like debian or ubuntu
## Why is this change important?
Can use the script  for the installation or update of the raspbian distribution

Update with cron the src git directory 
## How to test this PR locally?
Install searxng in the raspbian with the script
Update the src directory

sudo -H ./utils/searxng.sh install all
sudo -H ./utils/searxng.sh instance update

### CODE REVIEW

File: utils/lib.sh [hunk 1]
```diff
@@ -1686,7 +1686,7 @@ LXC_BASE_PACKAGES_fedora="bash git @development-tools python"
 LXC_BASE_PACKAGES_centos="bash git python3"
 
 case $DIST_ID in
-    ubuntu|debian) LXC_BASE_PACKAGES="${LXC_BASE_PACKAGES_debian}" ;;
+    ubuntu|debian|raspbian-*) LXC_BASE_PACKAGES="${LXC_BASE_PACKAGES_debian}" ;;
```
Comment (Bnyro) [inline utils/lib.sh:1689]:
```suggestion
    ubuntu|debian|raspbian) LXC_BASE_PACKAGES="${LXC_BASE_PACKAGES_debian}" ;;
```

Comment (Bnyro) [reply utils/lib.sh:1689]: We're only inspecting $DIST_ID here, not its version.

Comment (lannic) [reply utils/lib.sh:1689]: Yes i missed that

**Comment by return42** [comment]
Unfortunately, we cannot carry out any (automated) tests for raspbian, as we currently have no infrastructure for this.

Even if raspbian is based on debian, the packages are partly assembled differently; especially the uWSGI but also developer packages come to mind (the changes from this PR will most likely not be sufficient for all functions of the scripts) .. as long as we can't do any tests, there is always the danger that the distros will be affected by changes to the shell scripts as part of unwanted side effects.

Our long-term goal is to completely remove the installation scripts from the SearXNG core (I am already working on an alternative, but as I rarely have time for this, it may be a long time before I can present something to you).

When we have outsourced the installation one day, I can also imagine that we will support other, additional distributions.  Until then, I don't want to increase the complexity and the associated maintenance effort of the shell scripts here in the SearXNG core any further.

I know that this is not a satisfactory solution, but unfortunately I can't offer more at the moment ... so please understand if I close this PR.

-----

Related: 

- https://github.com/searxng/searxng/pull/508#issuecomment-971562552
- https://github.com/searxng/searxng/discussions/2897
- https://github.com/searxng/searxng/issues/2033#issuecomment-1363930129

**Pull request closed without merging**